numpy/lib/function_base.py

```diff
@@ -1840,13 +1840,13 @@ def gradient(f, *varargs, **kwargs):
         return outvals
 
 
-def diff(a, n=1, axis=-1):
+def diff(a, n=1, axis=-1, prepend=None, append=None):
```

Comment:
Although I guess that's achievable with `np.array(None)`. Perhaps just a remark in the notes section would be sufficient